tests/test_repair.py

```diff
@@ -0,0 +1,30 @@
+import geopandas
+import maup
+
+# These tests are losely based off the test_example_case in test_prorate.py
+
+def test_example_close_gaps_repair_MI():
+    shp = geopandas.read_file("zip://./examples/MI.zip") # MI shapefile
```

Comment:
Stupid question: the new examples (which are ~16 MB) aren't part of the `pip` distribution, yes?

Reply:
Good question -- nope! I confirmed this by looking at the tarball here: https://pypi.org/project/maup/#files.